R/drive_mime_type.R

```diff
@@ -10,7 +10,7 @@
 #'   * File extensions, such as "pdf", "csv", etc.
 #'   * MIME types accepted by Google Drive (these are simply passed through).
 #'
-#' @description If no input is provided, function returns the full table used
+#' @description If `type = all`, function returns the full table used
```

Comment:
I don't think we can do it this way. Because `"all"` is character input that, in theory, could be legitimate input. Now, I grant you that `"all"` is not a file extension nor a human-friendly nickname nor an actual MIME type. And yet I still don't think we should do this. I will think on it.